index.js

```diff
@@ -232,10 +236,10 @@ async function HandleMessage(message, sender, respond) {
   cmd = cmd.toLowerCase();
  
   // removes the bang at the beginning
-  const command = cmd.slice(1)
-  if (Object.keys(dummyMapping).includes(command)) {
-    respond(dummyMapping[command](sender, args))
-  }
+  let command = cmd.slice(1)
+  const fallback = () => `!${command} is not a valid command`
```

Comment:
I think the bot not having a response (instead of printing that the command is not valid) on any other command would be the way to go, since people don't want chat to be spammed and also might have other bots who answer on other commands.

Reply:
The fallback could be a function that gets passed the `response` function (see other comment on line 242) that will then not do anything (no-op).